backend/presentation/views.py

```diff
@@ -146,7 +146,10 @@ def getInboxLike(request, author_id):
     author = Author.objects.get(id=au_id)
     inbox = Inbox.objects.get(author=author)
     like_list = []
+    print(inbox.items)
     for each in inbox.items:
         if each["type"] == "Like":
             like_list.append(each)
+    print("-------")
+    print(like_list)
```

Comment:
```suggestion
    print("-------")
    print(like_list)
```
Delete debugging codes